pod.go

```diff
@@ -235,6 +235,13 @@ type PodStatus struct {
 	ContainersStatus []ContainerStatus
 }
 
+// ShimInfo gathers information needed by a shim, returned from a proxy.
+type ShimInfo struct {
+	Token string
+	IP    string
+	Port  string
+}
```

Comment:
Yes you're right. But in case I use a token list, how can I know (from runtime implementation) how to differentiate the tokens ?